src/assets/messages.json

```diff
@@ -3350,7 +3350,7 @@
     "set-group-membership-groups-required": "At least one group should be specified",
     "date-must-be-selected": "A specific date must be selected",
     "input-must-be-a-number": "Input must be a number",
-    "numeric-block-must-be-selected": "A block must be selected",
+    "numeric-block-must-be-selected": "An (existing) block must be selected",
```

Comment:
With this updated wording I've tried to cover both:
- that something should be selected
- and that it should be a block existing on the flow

Reply:
Alter to `An existing block must be selected` as per discussion.